e2e/deno-e2e/tests/deno-integrated.spec.ts

```diff
@@ -484,5 +484,214 @@ console.log(${fnName}())`
+      writeFileSync(join(tmpProjPath(), 'apps', bundlerAppName, 'assets/nx-ignore.hbs'), 'IGNORE ME');
+      writeFileSync(join(tmpProjPath(), 'apps', bundlerAppName, 'assets/a/b/nested-ignore.hbs'), 'IGNORE ME');
+
+      const project = readJson(`apps/${bundlerAppName}/project.json`);
+      project.targets.build.options.assets = [
+        `apps/${bundlerAppName}/*.md`,
+        {
+          input: `apps/${bundlerAppName}/assets`,
+          glob: '**/*.hbs',
+          output: 'assets',
+          ignore: ['ignore.hbs', '**/nested-ignore.hbs'],
+        },
+        'LICENSE',
+      ];
+      updateFile(`apps/${bundlerAppName}/project.json`, JSON.stringify(project));
+
+      const result = await runNxCommandAsync(`build ${bundlerAppName}`);
+      expect(result.stdout).toContain(
+        `Successfully ran target build for project ${bundlerAppName}`
+      );
+      expect(() => checkFilesExist(
+        `dist/apps/${bundlerAppName}/main.js`,
+        `dist/apps/${bundlerAppName}/LICENSE`,
+        `dist/apps/${bundlerAppName}/README.md`,
+        `dist/apps/${bundlerAppName}/assets/test1.hbs`,
+        `dist/apps/${bundlerAppName}/assets/test2.hbs`
+      )).not.toThrow();
+      expect(workspaceFileExists(`dist/apps/${bundlerAppName}/assets/ignore.hbs`)).toBeFalsy();
+      expect(workspaceFileExists(`dist/apps/${bundlerAppName}/assets/git-ignore.hbs`)).toBeFalsy();
+      expect(workspaceFileExists(`dist/apps/${bundlerAppName}/assets/nx-ignore.hbs`)).toBeFalsy();
+      expect(workspaceFileExists(`dist/apps/${bundlerAppName}/assets/a/b/nested-ignore.hbs`)).toBeFalsy();
+    }, 120_000);
+
+    it('should create deno lib', async () => {
+      await runNxCommandAsync(`generate @nx/deno:lib ${bundlerLibName}`);
+      expect(readJson(`import_map.json`).imports).toEqual(
+        expect.objectContaining({
+          [`@proj/${bundlerLibName}`]: `./libs/${bundlerLibName}/mod.ts`,
+        })
+      );
+      expect(workspaceFileExists(`libs/${bundlerLibName}/mod.ts`)).toBeTruthy();
+      expect(
+        workspaceFileExists(`libs/${bundlerLibName}/src/${bundlerLibName}.test.ts`)
+      ).toBeTruthy();
+      expect(
+        workspaceFileExists(`libs/${bundlerLibName}/src/${bundlerLibName}.ts`)
+      ).toBeTruthy();
+    }, 120_000);
+
+// Fixed by https://github.com/nrwl/nx-labs/pull/311
+//     it('should be able to use import alias of lib in app for build', async () => {
+//       const fnName = names(bundlerLibName).propertyName;
+//       updateFile(
+//         `apps/${bundlerAppName}/src/main.ts`,
+//         `import { ${fnName} } from '@proj/${bundlerLibName}'
+
+// console.log(${fnName}())`
+//       );
+
+//       const result = await runNxCommandAsync(`build ${bundlerAppName}`);
+//       expect(result.stdout).toContain(
+//         `Successfully ran target build for project ${bundlerAppName}`
+//       );
+//       expect(workspaceFileExists(`dist/apps/${bundlerAppName}/main.js`)).toBeTruthy();
+//     }, 120_000);
```

Comment:
is there a reason why this is commented out? I think this is already tested above.

Reply:
This test is specifically for the deno_emit bundler. It currently fails as deno_emit is broken (existing bug, not new to this PR). I have opened a PR to fix that bug separately https://github.com/nrwl/nx-labs/pull/311